install/update.php

```diff
@@ -4097,7 +4097,7 @@ function do_printer_migration($migration) {
       * Modify displaypreference for PluginFusioninventoryPrinterLog
       */
       $pfPrinterLogReport = new PluginFusioninventoryPrinterLog();
-      $a_searchoptions = $pfPrinterLogReport->getSearchOptions();
+      $a_searchoptions = $pfPrinterLogReport->getSearchOptionsNew();
```

Comment:
In next lines, we search if _$a_searchoptions['the_id']_ exist, but with new system, must loop on all and check if _['id'] == 'the_id'_

Reply:
Hi, I've updated that check, please review again that part.